.gitignore

```diff
@@ -61,3 +61,5 @@ bash.exe.stackdump
 package-lock.json
 
 !/dist/package.json
+/.vs
+/demo/.angular/cache/13.2.6
```

Comment:
This `.gitignore` line only seems relevant to angular 13, not angular 12.

Reply:
That is correct.  We can probably kill this one.